src/main/cpp/fielddata/FieldElementCalculator.h

```diff
@@ -55,6 +55,19 @@ class FieldElementCalculator
     void CalculateCenters(robin_hood::unordered_map<FieldConstants::FIELD_ELEMENT, frc::Pose3d> &fieldConstantsPoseMap);
     frc::Pose3d AverageHexagonPose(frc::Pose3d &pose1, frc::Pose3d &pose2, frc::Pose3d &pose3, frc::Pose3d &pose4, frc::Pose3d &pose5, frc::Pose3d &pose6);
 
+    static constexpr units::length::inch_t m_XdistanceProcessors{16.0};
```

Comment:
It is preferred to name these variables with the first character after the underscore being lower case.  So, m_yDistanceProcessors, for instance.